src/test/java/seedu/address/model/person/EmailTest.java

```diff
@@ -13,20 +13,14 @@ public void constructor_null_throwsNullPointerException() {
         assertThrows(NullPointerException.class, () -> new Email(null));
     }
 
-    @Test
-    public void constructor_invalidEmail_throwsIllegalArgumentException() {
-        String invalidEmail = "";
-        assertThrows(IllegalArgumentException.class, () -> new Email(invalidEmail));
-    }
-
     @Test
     public void isValidEmail() {
         // null email
         assertThrows(NullPointerException.class, () -> Email.isValidEmail(null));
 
-        // blank email
-        assertFalse(Email.isValidEmail("")); // empty string
-        assertFalse(Email.isValidEmail(" ")); // spaces only
+        // blank email now valid as email is optional
```

Comment:
minor nit again but I feel like since comments are written for others to read, maybe something like "blank email, valid as email is optional" would be clearer? Since someone reading this code in the future may not know that previously, the email was compulsory

Reply:
Agreed! Left a similar comment in your Address PR #74 earlier as well